linkerd/core/src/main/scala/io/buoyant/linkerd/Linker.scala

```diff
@@ -74,7 +77,8 @@ object Linker {
     namers: Option[Seq[NamerConfig]],
     routers: Seq[RouterConfig],
     tracers: Option[Seq[TracerConfig]],
-    admin: Option[AdminConfig]
+    admin: Option[AdminConfig],
+    metrics: Option[Seq[MetricsExporterConfig]]
```

Comment:
bikeshed:  It seems like a _metrics_ section might have some other things (like, configuration that controls how metrics are _recorded_ in addition to exported.  Does it make sense to structure this as:

```yaml
metrics:
  exporters:
  - ... 
```